fileB

```diff
@@ -0,0 +1,13 @@
+Z
+
+Y
+
+X
+
+W
```

Comment:
change to a number as well

Reply:
did this:
[4db5968](https://github.com/Arithmetics/git-testing/commit/4db59680c4e95a18584b77fdd0cb6e9147854b28)